scripts/game.js

```diff
@@ -0,0 +1,64 @@
+var rows = 50;
+var cols = 100;
+
+var playing = false;
+
+// define grid using 'new' operator to create an instance of the grid object
+var grid = new Array(rows);
+var nextGrid = new Array(rows);
+
+var timer;
+var reproductionTime = 100;
+
+
+
+function buildGrid() {
+  for (var i = 0; i < rows; i++) {
+    grid[i] = new Array(cols);
+    nextGrid[i] = new Array(cols);
+  }
+}
+function loadGrid() {
+    buildGrid();
+}
+
+// click handler for when user clicks cell on grid and turns cell "on" or "off"
+function clickHandler() {
+
+    // loop through array with class of "on" and set to "off" when clicked and vice versa
+
+
+// Create game board grid
+function createGrid() {
+  var gridBoard = document.getElementById("grid-container");
+  console.log("grid maybe?", gridBoard);
+  var grid = document.createElement("grid");
+
+//   loops through
+  for (var i = 0; i < rows; i++) {
+    var gridRow = document.createElement("gridRow");
+    for (var j = 0; j < cols; j++) {
+      //
+      var cell = document.createElement("gridCell");
```

Comment:
This is similar to the previous comment. I do not believe you can use "gridRow" and "gridCell". I believe you would have to use "div" and give them the class "gridRow" and "gridCell".